### PR TITLE
Bug/kill selenium process

### PR DESCRIPTION
**Pull request by gcyoung1**
I fixed the test_session_browser_penncnp test to account for the fact that connect_penncnp returns a generator. I also changed the list_running_process command so that it doesn't truncate the command names [this was causing existing selenium processes to not be detected].
I discovered that there is already a function, disconnect_penncnp, which kills the selenium process. I fixed the way that this function reports errors when calling kill, but this shouldn't affect the unkilled process problem. This function is called when the penncnp browser object goes out of scope. When I added a test_stale_penncnp test which makes two penncnp sessions, the test passed, suggesting that disconnect_penncnp is correctly called. When I wrote a version of this test to open a second session while the other session is still in scope, the "Error: sessions with display " + display + " are already \
running ! Please execute 'kill -9 "" error _did_ arise, as you would expect, but this is clearly not the cause of the  get_results_selenium errors.
In conclusion: I'm not sure why the get_results_selenium error arose. This PR just fixes the tests.

### CODE REVIEW

File: tests/test_session.py [hunk 1]
```diff
@@ -295,14 +295,17 @@ def penncnp_cleanup(session):
...
-    wait = session.initialize_penncnp_wait()
-    assert session.get_penncnp_export_report(wait)
+        wait = session.initialize_penncnp_wait()
+        assert session.get_penncnp_export_report(wait), "Error: could not export report."
 
+def test_stale_penncnp(slog, config_file, session, config_test_data, penncnp_cleanup):
+    project = 'browser_penncnp'
+    with session.connect_server(project, True) as server:
+        with session.connect_server(project, True) as server2:
+            assert server2 != None, "Error: could not create second server"
```
Comment (jimklo) [inline tests/test_session.py:308]: @gcyoung1 A couple questions:

1. would an `assert server == None` after line 308 or some other check to verify that `server` was terminated?  I'm assuming that when entering the context of `server2`, `server` would get terminated.  If yes, could you modify to validate that `server` has been killed? 
2. In a similar vein, should we verify that after exiting the `server2` context that selenium has exited?

**Review by jimklo — CHANGES_REQUESTED** [review]
@gcyoung1 Please address the questions, and resolve the merge conflicts, and then I can review again.

Thanks!

File: tests/test_session.py (hunk 1 above)
Comment (gcyoung1) [reply tests/test_session.py:308]: @jimklo 
1. Server is not terminated upon entering server2's context. Currently this test just fails, since the second attempt at creating a server detects the already running selenium process. We would need to remove the test for existing selenium processes to allow the second one to spawn. I'm not sure if that's something we even want to allow. So maybe I should just remove this test.
2. I think it would be better to make a separate test. Just create a single session then verify that the selenium process has been killed after we exit its context.

**Review by jimklo — APPROVED** [review]
Looks good
